Admin/SharedBlockAdmin.php

```diff
@@ -88,15 +88,15 @@ protected function configureFormFields(FormMapper $formMapper)
     }
 
     /**
-     * {@inheritDoc)
+     * {@inheritDoc).
```

Comment:
Maybe create a seperate PR for the phpdoc changes as they are not relevant for this feature